src/core_landice/Registry.xml

```diff
@@ -1313,6 +1415,14 @@ is the value of that variable from the *previous* time level!
 	       description="generic work array with dimensions of (nCells)"
 	       persistence="scratch"
 	       />
+      <var name="meanFlowParamA" type="real" dimensions="nCells" units="Pa^{-3} s^{-1}"
+	       description="depth averaged flow parameter A"
+	       persistence="persistent"
```

Comment:
It looks like I changed this to allow it to be output for debugging purposes when I was trying to move the stress calcs to the vel mod. So it can probably just be changed back to 'scratch'. I can sign up for this as well.

Reply:
done